github/ipfs.yml

```diff
@@ -4877,7 +5064,6 @@ teams:
         - laurentsenta
         - marten-seemann
         - petar
-        - reidlw
```

Comment:
@galargh :  We want to have @reidlw in the list I believe (or is this handled somewhere else)?

Reply:
This change was introduced via sync - https://github.com/ipfs/github-mgmt/pull/47/commits/f7ed8b0b5d1b1edb0469928677f0f64fd3d80fe2

That's because the invites weren't accepted in time, see:
<img width="977" alt="Screenshot 2022-09-06 at 12 42 15" src="https://user-images.githubusercontent.com/6688074/188615423-1bf9c3bd-576e-4126-8eac-ee7ea3faf81c.png">

I created the second one here: https://github.com/ipfs/github-mgmt/pull/44#issuecomment-1225601568